src/components/TodoContainer.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
Please Kindly disable `eslint` on your codebase. `eslint` enables you to go through the best practices on javascript code. refer [wikipedia](https://en.wikipedia.org/wiki/ESLint) 👍